pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -472,6 +472,7 @@ type ClusterStatus struct {
 	CertificateAuthorityData []byte `json:"certificateAuthorityData,omitempty"`
 	ARN                      string `json:"arn,omitempty"`
 	StackName                string `json:"stackName,omitempty"`
+	EKSCTLCreated            string `json:"eksctlCreated,omitempty"`
```

Comment:
Maybe we can have a new type for this with true false and unknown? Or at least `const string`s

Reply:
Good idea!